### PR TITLE
Sprint 2/issue 14#create sign in screen

### LINKED ISSUE #14: Create Signin Screen
## DESCRIPTION

 Design a visually appealing login screen where users will input their authentication details. This screen should align with the overall aesthetic of the application. The current focus is solely on creating the visual elements, not implementing the functionality.
 

- [x]  Create graphical assets needed for the login screen, such as icons, buttons, background images, or animations.
- [x] Develop the login screen in the application based on the design, ensuring the visual elements align with the overall application aesthetic.

### PR DESCRIPTION
**Pull request by Antonio3veces3**
# Description
In this branch we create the Sign in Screen, where the user has two options to sign in, their can sign in with Google and Facebook. 
This code is only design, doesn't have functionality.

What did you implemented/Why did you implemented this:

- Create the design of SignIn Screen
- Create buttons to sign in with Google
- Create buttons to sign in with Facebook
- Add icon of Google and Facebook into the buttons.

## Screenshots

In this image we can se the finished Sign In screen.
This is the result:
signInScreen

### CODE REVIEW

**Review by Lizzflores — APPROVED** [review]
LGTM

**Review by AlbertoG22 — APPROVED** [review]
Nice 🚀

File: CtasApp/yarn.lock [hunk 1]
```diff
@@ -3197,11 +3212,24 @@ eslint-visitor-keys@^2.1.0:
...
+  resolved "https://registry.npmjs.org/eslint-visitor-keys/-/eslint-visitor-keys-3.4.2.tgz"
+  integrity sha512-8drBzUEyZ2llkpCA67iYrgEssKDUu68V8ChqqOfFupIaG/LCVPUT+CoGJpT77zJprs4T/W7p07LP7zAIMuweVw==
+
+eslint-visitor-keys@^3.4.2:
   version "3.4.2"
   resolved "https://registry.npmjs.org/eslint-visitor-keys/-/eslint-visitor-keys-3.4.2.tgz"
   integrity sha512-8drBzUEyZ2llkpCA67iYrgEssKDUu68V8ChqqOfFupIaG/LCVPUT+CoGJpT77zJprs4T/W7p07LP7zAIMuweVw==
 
+=======
+>>>>>>> main
```
Comment (BrandonVG) [inline CtasApp/yarn.lock]: Be careful, you have to remove those characters that a conflicting merge left behind, in this file its not a big problem, but be careful if this happens in an important file

**Review by BrandonVG — APPROVED** [review]
image

**Comment by J3fte** [comment]
Buen diseño

**Pull request merged**